src/js/media-container.js

```diff
@@ -348,6 +350,8 @@ class MediaContainer extends window.HTMLElement {
     // Unhide for keyboard controlling
     this.addEventListener('keyup', (e) => {
       scheduleInactive();
+
+      this.handleKeyupEvent(e);
```

Comment:
suggestion(blocker): We should use the keydown + keyup strategy here or similar to filter out false positives while meta keys are pressed https://github.com/muxinc/media-chrome/blob/main/src/js/media-chrome-button.js#L108-L126 . If/when we add custom mappings, we'd still likely want "exact matches" even for multi-key support